src/lxc/confile.c

```diff
@@ -222,9 +222,9 @@ static struct lxc_config_t config_jump_table[] = {
 	{ "lxc.mount.fstab",                     set_config_mount_fstab,                      get_config_mount_fstab,                      clr_config_mount_fstab,                     },
 	{ "lxc.namespace.clone",                 set_config_namespace_clone,                  get_config_namespace_clone,                  clr_config_namespace_clone,                 },
 	{ "lxc.namespace.keep",                  set_config_namespace_keep,                   get_config_namespace_keep,                   clr_config_namespace_keep,                  },
+	{ "lxc.namespace.share",                 set_config_namespace_share,                  get_config_namespace_share,                  clr_config_namespace_share,                 },
 	{ "lxc.namespace.time.offset.boot",	 set_config_namespace_time_offset_boot,       get_config_namespace_time_offset_boot,       clr_config_namespace_time_offset_boot,      },
 	{ "lxc.namespace.time.offset.monotonic", set_config_namespace_time_offset_monotonic,  get_config_namespace_time_offset_monotonic,  clr_config_namespace_time_offset_monotonic, },
-	{ "lxc.namespace.share",                 set_config_namespace_share,                  get_config_namespace_share,                  clr_config_namespace_share,                 },
```

Comment:
Not sure what that's doing in this commit

Reply:
This looks like something that github is displaying weirdly because I'm sure not seeing that locally.